templates/snmpd.service-Debian.epp

```diff
@@ -0,0 +1,18 @@
+###
+### File managed by Puppet
+###
+[Unit]
+Description=Simple Network Management Protocol (SNMP) Daemon.
+After=network.target
+ConditionPathExists=/etc/snmp/snmpd.conf
+
+[Service]
+Environment="MIBSDIR=/usr/share/snmp/mibs:/usr/share/snmp/mibs/iana:/usr/share/snmp/mibs/ietf:/usr/share/mibs/site:/usr/share/snmp/mibs:/usr/share/mibs/iana:/usr/share/mibs/ietf:/usr/share/mibs/netsnmp"
+Environment="MIBS="
+Type=simple
+ExecStartPre=/bin/mkdir -p /var/run/agentx
+ExecStart=/usr/sbin/snmpd <%= $snmp::snmpd_options %>
```

Comment:
If we use this method, we will have to run 'systemctl daemon-reload' to reload units each time, then restart the service.

A simple wait is to use the `/etc/default/snmpd`. So,
please add in the `[Service]` section the line :
```
EnvironmentFile=-/etc/default/snmpd
```

And since the module profile the systend unit, we probably must have a look to this bug report:
https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=878828